tofu/geom/_def.py

```diff
@@ -164,7 +164,6 @@ def Plot_3D_plt_Tor_DefAxes(fs=None, wintit='tofu'):
     ax.set_xlabel(r"X (m)")
     ax.set_ylabel(r"Y (m)")
     ax.set_zlabel(r"Z (m)")
-    ax.set_aspect(aspect="equal", adjustable='datalim')
```

Comment:
Good idea,

However, I would recommend keeping the try / except, because 3d plotting in matplotlib is not very stable,
This plot, for instance, used to work fine until a matploltib update.
Given that 3d plotting is not very stable, I would recommend both solution:
* Try to make it work with your fix
* Keep the try / except as back-up safety in case it stops working for any reason at some point

Can you update your PR by adding the try / except with warning as in PR #311 ?

Reply:
I understand that `aspect="equal"` causes the error, but does the second parameter should also be removed?

Reply:
Good point, I'm going to try.

Reply:
No, it doesn't work.
```
ax.set_aspect(adjustable='datalim')
TypeError: set_aspect() missing 1 required positional argument: 'aspect'
```

Reply:
Ok, it bothers me though to just take out this line. @Didou09 : i'm under the impression that it worked in some cases, no? I have already used this feature of tofu to debug a camera. We could try to replace it with a hack like: https://stackoverflow.com/questions/13685386/matplotlib-equal-unit-length-with-equal-aspect-ratio-z-axis-is-not-equal-to

Reply:
It may have been a backwards incompatible change in matplotlib no?